src/components/annotations/draw.js

```diff
@@ -168,7 +168,6 @@ function drawOne(gd, index) {
     var font = options.font;
 
     var annText = annTextGroupInner.append('text')
-        .classed('annotation', true)
```

Comment:
This commit is reverted in ababc49 to make the tests added in https://github.com/plotly/plotly.js/pull/1683 :white_check_mark: